sklearn/ensemble/tests/test_forest.py

```diff
@@ -1466,8 +1467,29 @@ def check_warm_start_oob(name):
 
 
 @pytest.mark.parametrize("name", FOREST_CLASSIFIERS_REGRESSORS)
-def test_warm_start_oob(name):
```

Comment:
Looks like something went wrong with the last patch.

Can you add `test_warm_start_oob` back and move the body of the new function into `test_oob_not_computed_twice`?

Reply:
Sorry, I mistakenly deleted the wrong function. I've ran pytest to make sure and pushed the corrected edit.